Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -663,9 +663,10 @@ open class NavigationMapView: UIView {
      Updates `UserLocationStyle` to provided location.
      
      - parameter location: Location, where `UserLocationStyle` should be shown.
+     - parameter heading: Optional heading, which allows camera to move using heading instead of bearing for walking directions.
```

Comment:
“Bearing” is an Android-centric term we have to use internally because of MapboxNavigationNative and to some extent the map SDK, but it’s confusing to anyone who works directly with Core Location.

```suggestion
     - parameter heading: Optional heading, which allows camera to move using heading instead of course for walking directions.
```